facility-manager.html

```diff
@@ -0,0 +1,791 @@
+
+</li>
+
+</ol>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.insight</h3>
+
+<ol>
+
+<li>
+
+<p>Select the <span class="bold">Administrator</span> tab in the left hand pane.</p>
+
+<p>Select a group of which the user is already a member.</p>
+
+<p>Select the user from the group members.</p>
+
+<p>In the right-hand pane, select the new default group from the drop-down list.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/facilityManagerDefaultInsight.jpg" class="align-center" src="images/facilityManagerDefaultInsight.jpg" style="width: 100%;" />
+
+</li>
+
+</ol>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="new"></a>Creating groups and users</h2>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.web</h3>
+
+<ol>
+
+<li>
+<p class="first">Click on <span class="bold">Admin</span> in the top toolbar.</p>
+
+<p>Click on the <span class="bold">Groups</span> tab.</p>
+
+<p>Click on the <span class="bold">Add new Group</span> button.</p>
+
+<p>Fill in the details.</p>
+
+<p>Add group owners and members.</p>
+
+<p>Click to select the permissions level.</p>
+
+<p>Click <span class="bold">Save</span> to save the new group.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/facilityManagerGroupWeb.jpg" class="align-center" src="images/facilityManagerGroupWeb.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li>
+<p class="first">Click on the <span class="bold">Users</span> tab.</p>
+
+<p>Click on the <span class="bold">Add new User</span> button.</p>
+
+<p>Fill in the details.</p>
+
+<p>Add a group to make the user a member of it.</p>
+
+<p>Select the users default group in the drop-down list.</p>
+
+<p>Click <span class="bold">Save</span> the new user.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/facilityManagerUserWeb.jpg" class="align-center" src="images/facilityManagerUserWeb.jpg" style="width: 100%;" />
```

Comment:
Firefox looks fine font-wise though and the line isn't there on that either, just Safari (which I'm using an out-of-date version of so don't worry)